### ISSUE TITLE
Vertical nav - cannot tab to grouped items in Firefox

### ISSUE THREAD

**Issue by Jinnie** [vmware/clarity#5021]
## Describe the bug

In Firefox tabbing goes to next group instead of moving focus to child items in the group

## How to reproduce

1.  Use Firefox
2.  Go to https://clarity.design/documentation/vertical-nav/basic-structure/project-pokemon
3.  Scroll down to the Hierarchy example
4.  Click on a group to open it
5.  Try using Tab key to move focus to a child item.
6.  Verify that it skips all child items and goes to next group header.
7.   Repeat the above (1-6) in Chrome to verify that you can use Tab to focus child elements.

## Expected behavior

Should be able to use Tab to move focus to children

## Versions

**App**

- Clarity: v1 - v4

**Device:**

- Type: MacBook
- **Browser: Firefox**

**Comment by Jinnie** [comment]
Closing, to avoid duplication, as it is now tracked by our accessibility team.
